### DISCUSSION TITLE
Is it possible to generate a document from a base64?

### DISCUSSION THREAD

**Discussion by PPetterssonSO** [gotenberg/gotenberg#865] [Q&A]
Hi,
First of all, thanks for a great convert solution, top notch!

At the moment, we are converting files using a /libreoffice/convert endpoint and sending a bytearray as form-data
But, we have a scenario where we would like to send a base64 instead.

Is this possible somehow?

Regards
Pär

**Comment by gulien** [comment]
Hello @Nawkse 👋 

You may have to convert your base64 data into a bytearray on the client side 🤔 

I don't know if it's possible to send base64 directly though.
